cns/restserver/ipam.go

```diff
@@ -816,8 +823,15 @@ func (service *HTTPRestService) AssignAvailableIPConfigs(podInfo cns.PodInfo) ([
 
 	// Checks to make sure we found one IP for each NC
 	if len(ipsToAssign) != numOfNCs {
```

Comment:
this doesnt seem to be the right place for the check, shouldnt it be in line #856

Reply:
Line #856 will be too late as over here itself we determine if there are enough IPs present to allocate to the pod from each NC. If ips to assign are not sufficient we will return right here.